pkg/url/normalize.go

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-package database
+package url
 
 import "github.com/nlnwa/whatwg-url/url"
 
-func NormalizeUrl(u string) string {
+func Normalize(u string) string {
```

Comment:
Burde bruke ScopeCheck service sin normalizer for å sikre konsistent normalisering i alle komponenter, men det tar vi i en egen issue.